### PR TITLE
added offset

### PR DESCRIPTION
**Pull request by RiMikheev**
Added an offset to determine coordinates when searching for words in certain areas of image

### CODE REVIEW

File: src/pygats/recog.py [hunk 1]
```diff
@@ -263,8 +265,10 @@ def crop_image(img: Image, width: Optional[int] = 0, height: Optional[int] = 0,
                   crop_height * height,
                   crop_width * width + crop_width * factor,
                   crop_height * height + crop_height * factor)
+    x_offset = crop_coord[0]
+    y_offset = crop_coord[1]
     img_crop = img.crop(crop_coord)
-    return img_crop
+    return (x_offset, y_offset, img_crop)
```
Comment (ProjectMutilation) [inline src/pygats/recog.py:271]: extra braces

File: src/pygats/recog.py [hunk 2]
```diff
@@ -310,13 +316,14 @@ def find_text(img: Image, txt, skip=0, extend=False, one_word=False):
...
     """
-    img = find_crop_image(img, txt.area, extend=extend)
+    x_offset, y_offset, img = find_crop_image(img, txt.area, extend=extend)
     recognized = pytesseract.image_to_data(img, txt.lang).split('\n')
     lines = combine_lines(recognized, one_word)
     roi, found = ROI(-1, -1, -1, -1), False
     for pos, content in lines[1:]:
         if content.find(txt.content) != -1:
             print("Найден текст " + content)
+            pos = ROI(pos[0] + x_offset, pos[1] + y_offset, pos[2], pos[3])
```
Comment (ProjectMutilation) [inline src/pygats/recog.py:327]:
```
roi = ROI(pos[0] + x_offset, pos[1] + y_offset, pos[2], pos[3])
found = True
```
extra calling of assignment

**Review by ProjectMutilation — CHANGES_REQUESTED** [review]

File: src/pygats/recog.py (hunk 1 above)
Comment (RiMikheev) [reply src/pygats/recog.py:271]: fixed

**Comment by ProjectMutilation** [comment]
LGTM

**Pull request merged**